bia_export/bia_client_utils.py

```diff
@@ -55,14 +60,16 @@ def get_image_by_accession_id_and_relpath(accession_id: str, relpath: str):
         return images[0]
 
 
-def get_file_references_by_study_uuid(study_uuid: str):
+def get_file_references_by_study_uuid(
+    study_uuid: str,
+) -> list[api_models.FileReference]:
     file_references = rw_client.get_study_file_references(
         study_uuid, limit=20, apply_annotations=True
```

Comment:
Should `limit` be in the function declaration with a default value of 20, so that users can change this if necessary? Maybe also apply_annotations?

Reply:
I was thinking the same thing, but i did not want to make any functional changes whilst just moving the methods or adding types for ease of the review.